<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkflowDialog</class>
 <widget class="QDialog" name="WorkflowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>333</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="workflowPage">
      <layout class="QVBoxLayout" name="workflowPageLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="errorLabel"/>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="waitingPage"/>
     <widget class="QWidget" name="summaryPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="summaryLabel"/>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="summaryButtonBox"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
